chef/data_bags/crowbar/migrate/cinder/033_add_max_min_size_timeout_pool.rb

```diff
@@ -0,0 +1,13 @@
+def upgrade ta, td, a, d
+ a['min_pool_size'] = ta['min_pool_size']
```

Comment:
since you also submitted a backport to stoney, this code needs to check if the value exists already and then not overwrite it again from the template but preserve the original value (make the upgrade a noop then).